26-register_publication/creator-navigates-to-contributor-tab.feature

```diff
@@ -14,8 +14,76 @@ Feature: Creator navigates to Contributors tab
     And they see Save is enabled
 
   @1837
-  Scenario: Creator adds themselves to list of Authors
+  Scenario: Creator adds themselves to the list of Authors
     Given Creator begins registering a Registration in the Wizard
     When they navigate to the Contributors tab
-    And they click "Add me as author"
-    Then their Author identity is appended to list of Authors
+    And they see the "Add Author" Button
+    And they click "Add Author"
+    And they see the Author Search Dialog
+    And they click "Add me as Author"
+    Then their Author identity is added to the list of Authors
+
+  Scenario: Creator adds an Author to the list of Authors
+    Given Creator begins registering a Registration in the Wizard
+    When they navigate to the Contributors tab
+    And they see the "Add Author" Button
+    And they click "Add Author"
+    And they search for Author in the Author Search Dialog
+    And they select an Author identity
+    And they click "Add"
+    Then the selected Author identity is added to the list of Authors
+
+  Scenario: Creator adds an Author to the list of Authors for Reference Type Book, Monograph
```

Comment:
Er ikke dette det samme for de aller fleste? alle subtyper av tidsskrift, rapport, osv.
kanskje type skulle vært lagt inn via Examples?

Reply:
blir det rett med 3 scenario (add author, add editor, add supervisor), hvor examples sier hvilke typer det gjelder for

Reply:
Jo, default er author. Men for noen typer skal det _kun_ være author. Og for noen andre skal det være andre typer (en eller flere). Kunne laget examples, men føler det ble så rotete når det avhenger av Givens (hvilke subtyper man velger). Føler det blir mer eksplisitt å skrive scenario ut ifra valgt type (slik som vi har gjort ellers også). Uansett så er ikke disse featurene med author knyttet til noen Jira-oppgave pga de allerede er implementert.

Reply:
jaok, men kan jo bli en del scenarios da

Reply:
ja. stemmer for å samle disse når vi vet alle typene som skal ha hvilke roller